### ISSUE TITLE
send message with action tense is broken

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#409]
*Issue migrated from trac ticket [#125](http://colloquy.info/project/ticket/125)*

**type:** Defect | **component:** Colloquy (Mac) | **severity:** Major | **priority:** Normal | **version:** 2.0 (Mac) | **status:** closed | **resolution:** Works for Me | **keywords:** scripted actions
___
#### 2005-04-15 13:51:26: @brennanyoung created the issue

Steps to repro:

tell application "Colloquy" to send message "/nick testNick" with action tense

Expected result: nick gets changed

Actual result: "/nick testNick" is sent as chat text

This means it's impossible to run actions from applescript.
___
#### 2005-04-15 16:01:54: @xeenon changed resolution from ** to *wontfix*

___
#### 2005-04-15 16:01:54: @xeenon commented

This is not an action. That is a command. to send commands with AS you need to use "send raw command".

An example would be:

 send raw command "NICK testNick" to first connection

You can also do this for nick changes:

 set nickname of first connection to "testNick"

___
#### 2005-04-15 16:01:54: @xeenon changed status from *new* to *closed*

___
#### 2006-03-27 11:16:51: @brennanyoung commented

Ah, come on timothy. First of all, IRC users don't generally know raw event syntax, so it would be reasonable to expect something like /nick to be handled decently.

And it doesn't alter the fact that the following example fails 

tell application "Colloquy"
	tell first connection
		tell chat room 1
			send message "/me waves" with action tense
		end tell
	end tell
end tell

... and that's very definately an action, not a command. 

BTW if you leave out the slash, colloquy crashes!

___
#### 2006-03-27 11:16:51: @brennanyoung changed status from *closed* to *reopened*

___
#### 2006-03-27 11:16:51: @brennanyoung changed resolution from *wontfix* to **

___
#### 2006-03-27 19:04:55: @xeenon commented

That does not crash for me. "send message" is intended only for direct chat messages. We could consider looking for a / and parsing it as a command. However, a script might want to send a message that starts with a / (maybe a file path), how would we allow that? I would prefer to have a "send command" call. 

___
#### 2006-06-13 08:16:21: @xeenon changed status from *reopened* to *closed*

___
#### 2006-06-13 08:16:21: @xeenon changed resolution from ** to *worksforme*

___
#### 2006-06-13 08:16:21: @xeenon commented

Put simply, "with action tense" does not mean "run a /command". It means "send this text as an action", like "* xenon runs".

This is: send message "runs" with action tense

What you want is better described in #338. Closing this because action tense works as designed. Will keep #338 to track the rest of this.
